src/components/Reactions/Reactions.tsx

```diff
@@ -23,7 +24,6 @@ export const Reactions = React.memo(({ reactions, children, onClick }: Reactions
                         <Popup
```

Comment:
this was tooltip, in harmony we have tooltip, mb would be better to use tooltip component?